specs/001-docker-dev-stack/contracts/service-interfaces.md

```
@@ -0,0 +1,718 @@
+Service Name: postgres
+Image: postgres:15-alpine
+Container Name: wvwo-postgres-dev
+```
+
+**Port Mapping**:
+- Internal: `5432`
+- Host: `5432`
+- Protocol: TCP
+
+**Environment Variables** (all required):
+```yaml
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)